content/documentation/nodes/object/_index.md

```diff
@@ -6,4 +6,16 @@ chapter : true
 
 # Object
 
-This is the reference for object nodes in Animation Nodes.
+### Data Types
+
+- **Object** - It is an object that is available in the *Add* menu of the 3D viewport.
```

Comment:
Just call this a "Blender object", no need to mention the add menu.

Reply:
Okay.